Assets/Scripts/GameData/TerrainData.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;		// For dictionaries
+
+// Class holding game data for each terrain
+public class TerrainData {
+
+	public string shortName;
+	public string name;
+	public string description;
+	public string spritePath;
+
+	public TerrainData(KeyValuePair<string, object> terrain) {
+		Dictionary<string, object> terrainData = (Dictionary<string, object>)terrain.Value;
+		shortName = terrain.Key;
+		name = terrainData["DisplayName"].ToString();
+		if(terrainData.ContainsKey("Description")) {
+			description = terrainData["Description"].ToString();
+		}
```

Comment:
If check not necessary here.  Back-end will guarantee a key is returned.  Will also handle if it isn't set, so an empty string will be returned. (Other option would be to require this if check [and the corresponding else] for every single JSON field, which would over-complicate and force code duplication in the front end.)